apps/game_client/lib/game_client_web/controllers/page_html/home.html.heex

```diff
@@ -1,6 +1,6 @@
 <%= if assigns[:current_user_id] do %>
   <.form :let={f} for={%{}} action={~p"/"}>
-    <.input field={f[:character]} name="character" label="Select a Character" type="select" options={["muflus", "h4ck", "uma", "valtimer", "kenzu"]} value=""/>
+    <.input field={f[:character]} name="character" label="Select a Character" type="select" options={["muflus", "h4ck", "uma", "valtimer", "kenzu", "otix"]} value=""/>
```

Comment:
Please add otix in loadtest character list too 😄

Reply:
Whoops, done